pom.xml

```diff
@@ -221,8 +221,7 @@
                     <doctitle>Jakarta Bean Validation API ${project.version}</doctitle>
                     <windowtitle>Jakarta Bean Validation API ${project.version}</windowtitle>
                     <bottom>
-                        <![CDATA[Copyright &copy; ${project.inceptionYear}-{currentYear} <a href="http://redhat.com">Red Hat Inc.</a>  All Rights Reserved.
-                        Released under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache Software License 2.0</a>.]]>
+                        <![CDATA[This spec is licensed under the EFSL; this spec is based on material that is licensed under the Apache License, version 2.0.]]>
```

Comment:
@emmanuelbernard I'm not sure about this one. You said we should keep our copyright but @starksm64  asked for us to have this specific footer in the Javadoc.

Reply:
They way I understand is that the actual sources remain under the previous (Apache) license, but that the JavaDoc is published under EFSL.

Reply:
So the license and the copyright are two different things:
- keep the copyright Red Hat

And use this text (to align to what I put in the spec)

This specification is licensed under the <a href="https://www.eclipse.org/legal/efsl.php">Eclipse Foundation Specification License 1.0</a>; this specification is based on material that is licensed under the <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache Software License 2.0</a>.

Reply:
I am assuming that this bottom text is what is added to the javadoc and nowhere else.
Ideally, we would use a property to only activate this EFSL mention when doing a final spec release like we do swap the license-final file in the spec. But if that's too complicated let's not.

Reply:
I will see what I can do once I know what we can do on Eclipse CI.